src/openforms/data_removal/tests/test_tasks.py

```diff
@@ -57,6 +58,57 @@ def test_successful_submissions_correctly_deleted(self):
         with self.assertRaises(ObjectDoesNotExist):
             submission_to_be_deleted.refresh_from_db()
 
+    # GH #2632
```

Comment:
please tag these tests using `@tag("gh-2632")`, that makes it easier to track why they exist AND to run them using `src/manage.py  test src --tag=gh-2632`